parlai/crowdsourcing/projects/humaneval/single_model_eval/analysis/compile_results.py

```diff
@@ -0,0 +1,338 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import os
+from collections import defaultdict
+from datetime import datetime
+from typing import Any, Dict
+
+import pandas as pd
+from mephisto.data_model.worker import Worker
+
+import parlai.utils.logging as logging
+from parlai.crowdsourcing.tasks.model_chat.model_chat_blueprint import (
+    BLUEPRINT_TYPE,
+)  # noqa: F401  # For registering the blueprint
```

Comment:
Annoyingly I think you need to put this comment on line 16 for the lint to successfully ignore this import.

Reply:
ah good point, just changed